### PR TITLE
Fix use of labels for update mode

### PR DESCRIPTION
**Pull request by rbren**
This fixed a bug found by a user, who was trying to use a deployment's labels to set update mode.

### CODE REVIEW

**Comment by sudermanjr** [comment]
You'll need to wait for me to merge #203 so that the e2e tests will work. Also can you please sign the CLA (you shouldn't need to since you're in the org, but :shrug: )

**Comment by rbren** [comment]
Ahh thanks for the heads up, couldn't figure out what I'd broken

**Comment by sudermanjr** [comment]
> Ahh thanks for the heads up, couldn't figure out what I'd broken

The demo chart we install as part of the e2e test needed updating because their image disappeared from docker hub :-(

**Comment by sudermanjr** [comment]
@rbren would you mind rebasing this now that client-go is updated and the e2e tests are fixed?

**Comment by rbren** [comment]
Done. Was kinda tricky so worth taking another look, but I think I got it.

File: pkg/vpa/vpa.go
Comment (lucasreed) [inline pkg/vpa/vpa.go:338]:
```suggestion
			UpdateMode: vpaUpdateMode,
```

updateMode is undeclared. I'm only guessing at the name of the actual value based on what else is in this function. Please have a look and make sure.

**Review by lucasreed — CHANGES_REQUESTED** [review]

File: pkg/vpa/vpa.go
Comment (rbren) [reply pkg/vpa/vpa.go:338]: Whoops, thanks, fixed

**Review by sudermanjr — APPROVED** [review]

**Review by lucasreed — APPROVED** [review]
This is just a rubber stamp, but approving with CI passing.

**Pull request merged**
